shortlister/controller.py

```diff
@@ -91,14 +92,14 @@ def edit_criteria_select(self, k=None):
         self.view.view_selection_options(self.current_criterion)
         self.options = {
             str(i): (self.edit_score_confirm,s)
-            for i, s in enumerate(self.current_criterion.scores)
+            for i, s in enumerate(SCORE_AND_VALUE.keys())
         }
         self.options["c"] = (self.clear_score,f"Clear score: {self.current_criterion.name}")
 
     def edit_score_confirm(self, k=None):
         """Updates the selected score of previously select criteria."""
         self.view.view_update(
-            self.current_criterion.name, self.current_criterion.scores[int(k)]
+            self.current_criterion.name, list(SCORE_AND_VALUE.keys())[int(k)]
```

Comment:
Similarly, you don't need `.keys()` here - `list(a_dictionary)` returns the keys of the dictionary as a list. Ther are other places in this PR where you can make this simplification.